.ci/generate_test_report_lib_test.py

```diff
@@ -169,12 +208,13 @@ def test_no_tests_in_testsuite(self):
                         )
                     )
                 ],
+                [],
             ),
             dedent(
                 """\
                 # Foo
 
-                The build failed before running any tests.
+                The build failed before running any tests. Detailed information about the build failure could not be automatically obtained.
```

Comment:
This is pedantry, but apparently that's one of my talents so anyway - 

I would just say "Information about the build failure...". Since the level of detail is either 0, or whatever is in the build log. There's no spectrum of detail in between.

People will have to read the log file in this case anyway, let's not get their hopes up that the log file will be any more helpful :)

Reply:
Good point. Updated.